Cargo.toml

```diff
@@ -18,18 +18,18 @@ value = ["number-general/stream"]
 all = ["tokio-io", "value"]
 
 [dependencies]
-async-trait = "0.1"
-base64 = "0.21"
+async-trait = "0.1.78"
+base64 = "0.22"
 bytes = "1.5"
 destream = "0.7"
 futures = "0.3"
-number-general = { version = "0.11", optional = true }
+number-general = { version = "0.11.1", optional = true }
 pin-project = "1.1"
 tokio = { version = "1.35", features = ["io-util"], optional = true }
 uuid = "1.6"
 
 [dev-dependencies]
-number-general = { version = "0.11", features=["stream"] }
+number-general = { version = "0.11.1", features=["stream"] }
```

Comment:
updated these versions as an attempt to get [this minimal-versions test](https://github.com/DrewMcArthur/destream_json/pull/1/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R58) working, but ended up dropping it